CodeStarterCamp_Week2/main.swift

```diff
@@ -5,8 +5,35 @@
 //  Created by yagom.
 //  Copyright © yagom academy. All rights reserved.
 //
-
 import Foundation
 
-print("Hello, World!")
+func createLottoNumbers() -> [Int] {
+    var lottoNumbers: [Int] = []
+    var index: Int = 0
+
+    while index < 6 {
+        let randNum: Int = Int(arc4random_uniform(45)) + 1
+
+        if lottoNumbers.firstIndex(of: randNum) == nil {
+            lottoNumbers.append(randNum)
+            index = index + 1
+        }
+    }
+
+    return lottoNumbers
+}
+
+func confirmLottoNumbers(_ myLottoNumbers:[Int], _ createdLottoNumbers:[Int]) -> String {
+    let winNumbers: [Int] = myLottoNumbers.filter{createdLottoNumbers.contains($0)}
+    
+    if winNumbers.count > 0 {
+        return "축하합니다. 겹치는 번호는 \(winNumbers.map{String($0)}.joined(separator: ", ")) 입니다"
```

Comment:
```suggestion
        return "축하합니다. 겹치는 번호는 \(winNumbers.map{ String($0) }.joined(separator: ", ")) 입니다"
```
여기도 축약형을 풀어서 쓰는 연습을 해보는 것이 좋을 것 같습니다!

처음엔 고차함수에 익숙하지 않기 때문에 풀어서 쓰는 습관을 들이면, 나중에 축약형 코드를 읽을 때도 많은 도움이 됩니다.